test/process/test.js

```diff
@@ -167,10 +167,13 @@ describe('process', function () {
           }
         })
 
-    it('should process \'/pet/findByStatus\' correctly with in-line example query param',
+    it('should process \'/pet/findByStatus\' correctly with in-line example query param, with explicit exclusion',
         function (done) {
           try {
             var data = process(api, {
+              'customValues': {
+                query: { petName: 'your pet name here <3' }
```

Comment:
Can we make this test go to the deepest level (status code) in the customValues? That would make me feel better about the coverage of the lookup function you've added.